Jasmine/Common/Extensions/CGPoint+Arithmetic.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import UIKit
+
+extension CGPoint {
+
+    /// Subtracts another point to get a displacement vector.
+    ///
+    /// - Parameter other: the other point to get displacement.
+    /// - Returns: the resultant vector
+    func sub(_ other: CGPoint) -> CGVector {
```

Comment:
OK it might be confused with it returning `CGPoint`. Hmm I think using `CGVector(from: CGPoint, to: CGPoint)` might be clearer?